lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -28,6 +29,7 @@
 from xmodule.modulestore.tests.factories import check_mongo_calls
 from opaque_keys.edx.locations import SlashSeparatedCourseKey
 from opaque_keys.edx.locator import CourseLocator
+from courseware.url_helpers import get_redirect_url
```

Comment:
add import with alphabetical order: http://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement